### PR TITLE
Implement source rework to HTML client

### PR DESCRIPTION
**Pull request by DanielBallaSZTE**
Implementing the source command rework - along with display - to the html client too.

JerryScript-DCO-1.0-Signed-off-by: Daniel Balla dballa@inf.u-szeged.hu

### CODE REVIEW

File: jerry-debugger/jerry-client-ws.html [hunk 1]
```diff
@@ -1161,11 +1166,56 @@
...
+                         lastBreakpointHit.func.source.length - 1)
+    }
+
+    if (lastBreakpointHit.func.sourceName)
+      appendLog("Source: " + lastBreakpointHit.func.sourceName);
+
+    for (i = start; i < end; i++)
+    {
+      var str = (i+1).toString();
+      while (str.length < 4){
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: Move `{` to the next line.

File: jerry-debugger/jerry-client-ws.html [hunk 2]
```diff
@@ -1161,11 +1166,56 @@
...
+      var end = Math.min(lastBreakpointHit.line + line_num - 1,
+                         lastBreakpointHit.func.source.length - 1)
+    }
+
+    if (lastBreakpointHit.func.sourceName)
+      appendLog("Source: " + lastBreakpointHit.func.sourceName);
+
+    for (i = start; i < end; i++)
+    {
+      var str = (i+1).toString();
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: `(i + 1).`

File: jerry-debugger/jerry-client-ws.html [hunk 3]
```diff
@@ -1365,8 +1415,32 @@
       debuggerObj.sendExceptionConfig(args[2]);
       break;
 
+    case "source":
     case "src":
-      debuggerObj.printSource();
+      if (!args[2]){
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: Move `{` to the next line.

File: jerry-debugger/jerry-client-ws.html [hunk 4]
```diff
@@ -1365,8 +1415,32 @@
...
+      var line_num = debuggerObj.srcCheckArgs(args[2]);
+      if (line_num >= 0)
+      {
+        debuggerObj.printSource(line_num);
+        break;
+      }
+      break;
+
+    case "display":
+      if (!args[2]){
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: Move `{` to the next line.

File: jerry-debugger/jerry-client-ws.html [hunk 5]
```diff
@@ -1365,8 +1415,32 @@
...
+        appendLog("Non-negative integer number expected,\
+                  0 turns off this function");
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: 0 turns off the automatic source code display

**Comment by LaszloLango** [comment]
@DanielBallaSZTE Please fix the style issues

**Comment by DanielBallaSZTE** [comment]
As webide is going to replace the html debugger, I close this PR, since it is not necessary anymore.

**Comment by LaszloLango** [comment]
@zherczeg could you review it again?

File: jerry-debugger/jerry-client-ws.html [hunk 6]
```diff
@@ -852,6 +853,10 @@
                   + (breakpoint.at ? "at " : "around ")
                   + breakpointInfo
                   + breakpointToString(breakpoint));
+        if (debuggerObj.display)
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.html]: Newline before if.

**Comment by DanielBallaSZTE** [comment]
@zherczeg I've added the new line.

**Pull request merged**